src/styles/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product {
+  width: 200px;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  border: 1px solid white;
+  border-radius: 5px;
+
+  &__image {
+    margin: 32px 19px 0;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    color: $main-accent;
+    margin: 36px 16px 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    color: $secondary;
+    margin: 4px 16px 0;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
this is a default browser value, no needs to redeclare it. Remove at other cases as well